ftw/recipe/deployment/pack.py

```diff
@@ -73,6 +77,32 @@ def create_pack_script(recipe):
     return created_files
 
 
+
+SH_SUBCMD_DATE = '`date +%Y-%m-%dT%H:%M:%S%z`'
+SH_MAKE_PACK_LOG_ENTRY = ('echo '
+                          + SH_SUBCMD_DATE
+                          + ' "packed {text}"'
+                          + ' > {buildout_dir}/var/log/pack.log')
+
+
+def build_packed_log_entry_command(recipe, storage, blobstorage=None):
+    if storage == '1':
+        storage = 'main'
```

Comment:
@buchi What do you prefer as default storage? Should we use `Data` here since the file usually is named `Data.fs`?